### PR TITLE
removed some unused variables

### PR DESCRIPTION
**Pull request by wkotlarski**

### CODE REVIEW

File: meta/Decays.m [hunk 1]
```diff
@@ -766,7 +766,7 @@ would violate Ward identity (as at the level of dim 4
                       If[numIndices == 0 || dim <= 1,
                          result = result <> " {};\n";,
                          result = result <> " {{" <> ToString[indexVal] <>
-                                  StringJoin[Table[", 0", {i, 1, numIndices - 1}]] <> "}};\n";
+                                  StringJoin[ConstantArray[", 0", numIndices - 1]] <> "}};\n";
```
Comment (uukhas) [inline meta/Decays.m:769]: Didn't know, that this function exists. I think, that now I need to change some code as well :)

**Review by Expander — APPROVED** [review]

**Comment by Expander** [comment]
I wish my emacs would point me to such unused variables. :)

**Comment by uukhas** [comment]
I think, that soon enough we will have a nice code checker thing, which will help us significantly with our mathematica code.

**Review by uukhas — APPROVED** [review]

**Comment by wkotlarski** [comment]
For emacs and vim it's not so easy. But for [Visual Studio Code](https://github.com/WolframResearch/vscode-wolfram) and [Sublime](https://github.com/WolframResearch/Sublime-WolframLanguage) the are official Wolfram plugins. They seem promising.

**Pull request merged**